### DISCUSSION TITLE
Compare files, ignore folder structure

### DISCUSSION THREAD

**Discussion by stroodl** [WinMerge/winmerge#1207] [Q&A]
Hello,

I need to compare two folders full of photos. Each of the folders has different subfolder structure. How do I compare the folders on the file level, ignoring the path to the file? WinMerge marks each and every file as "left only" even though the files are on the right as well, just in a different subfolder.

I need WinMerge to go file by file and check whether the file exists in the other folder, ignoring the file structure. (I have several external HDDs with unsorted photos. I copied them to a NAS while also sorting the photos to appropriate subfolders. Now I want to make sure that no photos were left on the HDDs).

Thank you,

Ondra

**Comment by sdottaka** [comment]
Unfortunately, WinMerge doesn't have the ability to ignore the folder structure and compare files.

**Comment by stroodl** [comment]
Thank you. Could you possibly recommend me a tool that is capable of such a comparison?

**Comment by sdottaka** [comment]
WinMerge2011 seems to be able to ignore the structure of the folders and compare them.

https://github.com/datadiode/winmerge2011/

![image](https://user-images.githubusercontent.com/98126/155432562-f69a574a-5ebf-422c-b3bc-c67624265d6a.png)
